lib/archethic/self_repair.ex

```diff
@@ -48,17 +50,35 @@ defmodule Archethic.SelfRepair do
     # Before the first summary date, synchronization is useless
     # as no data have been aggregated
     if DateTime.diff(DateTime.utc_now(), summary_time) >= 0 do
-      start_date = DateTime.utc_now()
-      :ok = Sync.load_missed_transactions(date)
-      put_last_sync_date(start_date)
+      loaded_missed_transactions? =
+        :ok ==
+          0..@max_retry_count
+          |> Enum.reduce_while(:error, fn _, _ ->
+            try do
+              :ok = Sync.load_missed_transactions(date)
+              {:halt, :ok}
+            catch
+              _, _ -> {:cont, :error}
+            end
+          end)
+
+      if loaded_missed_transactions? do
+        Logger.info("Bootstrap Sync succeded in loading missed transactions !")
+      else
+        Logger.error(
+          "Bootstrap Sync failed to load missed transactions after max retry of #{@max_retry_count} !"
```

Comment:
Does it mean we aren't crashing the process if there is no success?

Reply:
no we keep it alive, and see if the self repair will work on the next cycle